.github/workflows/ci-github-actions.yml

```diff
@@ -0,0 +1,60 @@
+name: CI Github Actions
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+  pull_request:
+
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
do we even need the matrix? There will be always just one set to test right?

Reply:
I removed the matrix